assets/airports/lrop.json

```diff
@@ -0,0 +1,1411 @@
+            ["N44d59m58.889", "E25d58m35.842", "N44d59m55.592", "E25d59m11.922"],
+            ["N44d59m50.103", "E25d59m47.468", "N44d59m42.464", "E26d00m22.215"],
+            ["N44d59m32.730", "E26d00m55.904", "N44d59m20.975", "E26d01m28.287"],
+            ["N44d59m07.286", "E26d01m59.121", "N44d58m51.765", "E26d02m28.178"],
+            ["N44d58m34.526", "E26d02m55.244", "N44d58m15.698", "E26d03m20.116"],
+            ["N44d57m55.421", "E26d03m42.612", "N44d57m33.845", "E26d04m02.565"],
+            ["N44d57m11.130", "E26d04m19.828", "N44d56m47.445", "E26d04m34.274"],
+            ["N44d56m22.966", "E26d04m45.795", "N44d55m57.873", "E26d04m54.309"],
+            ["N44d55m32.354", "E26d04m59.753", "N44d55m06.596", "E26d05m02.088"],
+            ["N44d54m40.791", "E26d05m01.297", "N44d54m15.130", "E26d04m57.388"],
+            ["N44d53m49.803", "E26d04m50.391", "N44d53m24.996", "E26d04m40.359"],
+            ["N44d53m00.894", "E26d04m27.366", "N44d52m37.675", "E26d04m11.511"],
+            ["N44d52m15.510", "E26d03m52.911", "N44d51m54.563", "E26d03m31.704"],
+            ["N44d51m34.988", "E26d03m08.048", "N44d51m16.931", "E26d02m42.117"],
+            ["N44d51m00.524", "E26d02m14.105", "N44d50m45.889", "E26d01m44.218"],
+            ["N44d50m33.133", "E26d01m12.677", "N44d50m22.351", "E26d00m39.715"],
+            ["N44d50m13.622", "E26d00m05.576", "N44d50m07.010", "E25d59m30.510"],
+            ["N44d50m02.565", "E25d58m54.777", "N44d50m00.318", "E25d58m18.641"],
+            ["N44d50m00.287", "E25d57m42.366", "N44d50m02.472", "E25d57m06.222"],
+            ["N44d50m06.857", "E25d56m30.475", "N44d50m13.409", "E25d55m55.387"],
+            ["N44d50m22.080", "E25d55m21.218", "N44d50m32.806", "E25d54m48.220"],
+            ["N44d50m45.509", "E25d54m16.636", "N44d51m00.093", "E25d53m46.701"],
+            ["N44d51m16.452", "E25d53m18.634", "N44d51m34.465", "E25d52m52.643"],
+            ["N44d51m54.000", "E25d52m28.921", "N44d52m14.911", "E25d52m07.644"],
+            ["N44d52m37.044", "E25d51m48.969", "N44d53m00.237", "E25d51m33.036"],
+            ["N44d53m24.316", "E25d51m19.963", "N44d53m49.106", "E25d51m09.847"],
+            ["N44d54m14.421", "E25d51m02.765", "N44d54m40.076", "E25d50m58.770"],
+            ["N44d55m05.879", "E25d50m57.892", "N44d55m31.641", "E25d51m00.140"],
+            ["N44d55m57.169", "E25d51m05.498", "N44d56m22.276", "E25d51m13.927"],
+            ["N44d56m46.775", "E25d51m25.367", "N44d57m10.485", "E25d51m39.732"],
+            ["N44d57m33.229", "E25d51m56.918", "N44d57m54.838", "E25d52m16.798"],
+            ["N44d58m15.154", "E25d52m39.226", "N44d58m34.024", "E25d53m04.035"],
+            ["N44d58m51.309", "E25d53m31.042", "N44d59m06.879", "E25d54m00.047"],
+            ["N44d59m20.621", "E25d54m30.835", "N44d59m32.431", "E25d55m03.177"],
+            ["N44d59m42.221", "E25d55m36.834", "N44d59m49.920", "E25d56m11.555"],
+            ["N44d59m55.469", "E25d56m47.082", "N44d59m58.827", "E25d57m23.151"],
+
```

Comment:
Please remove empty lines in the video map

Reply:
fixed both